src/js/capture.js

```diff
@@ -181,16 +202,42 @@ const capture = {
   // capture first party requests
   async sendFirstParty(tabId, changeInfo, tab) {
     const documentUrl = new URL(tab.url);
-    if (documentUrl.hostname
-        && tab.status === 'complete' && await this.shouldStore(tab)) {
+    // Greenbeam. Checking whether a given website is hosted by renewables.
+    // Note that for first parties it may take a wahile to load.
+    const greenStatus = await checkGreenStatus(documentUrl.hostname).then(data => {
```

Comment:
As above - while you can mix `async/await` with the `then` stuff, it's better to stick to one style if you can, and the rest of the this code base seems to use the `async/await` approach

Reply:
Yup, fixing this too!